packages/xrpl/snippets/src/bridgeTransfer.ts

```diff
@@ -75,6 +70,7 @@ async function bridgeTransfer(): Promise<void> {
       parseInt(bridgeData.MinAccountCreateAmount as string, 10) * 2
     ).toString(),
   }
+  console.log(fundTx)
```

Comment:
mean to have this?

Reply:
It's a snippet, so it's not a problem, but it's not necessary.